dali_mqtt_daemon.py

```diff
@@ -112,6 +112,20 @@ def dali_scan(driver, max_range=4):
     return lamps
 
 
+def display_lamps_summary(max_range, lamps_dicts):
+    logger.info(
+        "Found {} lamps from excepted {}, scanned in range 0-{}".format(
```

Comment:
Don't use format() in logging (run pylint to see the advice)

Reply:
Done

Reply:
@dgomes If this will be in separate function it will be easier to mainaint and read code.
It's easier to read good function and guess what is the purpose of the function than read the code.

Reply:
this functionality is currently trivial and requires an extra loop

Reply:
I agree., the loop isn't nice.